servicetalk-encoding-api/src/main/java/io/servicetalk/encoding/api/ContentCodecBuilder.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.encoding.api;
+
+/**
+ * Builder for {@link ContentCodec}.
+ */
+public interface ContentCodecBuilder {
+
+    int DEFAULT_MAX_ALLOWED_DECOMPRESSED_PAYLOAD = 16 << 20; //16MiB
```

Comment:
Consider moving it out of a public interface. `DefaultContentCodecBuilder` can be a good home for this private constant.